docs/user/guides/patching_agama_live.md

```diff
@@ -0,0 +1,114 @@
+
+Agama relies on the Driver Update Disk but it ignores some parts, like the
+[scripts](#running-scripts) or the "dist" (the `sle16` part in `linux/suse/x86_64-sle16`).
+
+So when we are talking about the content of the update, we will be referring to the files below
+`linux/suse/ARCH-*` (where `ARCH` is the architecture, like `x86_64`). Bear in mind that there might
+be multiple updates in a single file, each with a numeric prefix (`0/linux/suse/x86_64-sles16`,
+`1/linux/suse/x86_64-sles16`, etc.).
+
+:::
+
+## Applying an update
+
+To load an update, you must specify the URL using the `inst.dud` kernel option. You can set this
+option multiple times and each URL will be processed in the given order. About the URL formats, they
+are described in the [Agama URLs](/user/reference/urls) section.
+
+Updates are applied by a [dracut](https://github.com/dracut-ng/dracut-ng) module before Agama's live
+system boots so it gives us the chance to modify almost anything from the live system.
+
+## Features
+
+There are several things you can do with an update. In a nutshell, it is possible to:
+
+- Modify the Agama live image. It allows applying bug fixes to Agama or any related tool, changing
+  the configuration of the live image, etc.
+- Add packages to the installation media. You could install a patched version of a package instead
+  of the one included in the repositories.
+- Add or update kernel modules. It enables you to use an updated kernel module (unloading the old
+  one automatically) or adding your own modules.
+
+If you are familiar with Driver Update Disks, you might be wondering how to run your own scripts.
+The [Running scripts](#running-scripts) section explains how you do it.
+
+### Modifying the image
+
+You can modify the Agama live image using an RPM package or a Driver Update Disk. In the former
+case, the package is unpacked to the live image without running any script. If you prefer a Driver
+Update Disk, the files under the
+[inst-sys directory](https://ftp.suse.com/pub/people/hvogel/Update-Media-HOWTO/Update-Media-HOWTO.html#id_inst_sys)
+will be copied to the live image.
+
+### Adding packages for installation
+
+If you want to replace a package (or add a new one), you can place them under the `install`
+directory of your Driver Update Disk. Those packages will be copied to a special repository that
+Agama will use during installation. That repository will have higher priority than the product
+repositories, so those packages will be preferred.
+
+This purpose of this feature is replacing existing packages with updated or patched versions. If you
+want to add packages that are not in the distribution, you might need to tell Agama about those
+packages.
+
+### Adding or updating kernel modules
+
+The `modules` directory of your Driver Update might contain a set of kernel modules. Agama will copy
+them to the live image. If a `module.order` file is found, Agama will unload the modules listed in
```

Comment:
'module.order' suggests that the list is order dependent ... is that so? Should be explained.